src/reframe.js

```diff
@@ -21,8 +21,8 @@ export default function reframe(target, cName) {
   }
 }
 
-if (window.$) {
```

Comment:
@mohandere I'll be merging something to support your request as soon as possible. Thanks for you note! 🙏